### PR TITLE
Pulga fix backend seeding

### PR DESCRIPTION
**Pull request by pulgamecanica**
***

### About typeorm-seeding library :cry: 

**_typeorm-seeding library has been discontinued, you should be using other options._**

***

##### Here I suggest an implementation of the new library :new: 
[https://www.npmjs.com/package/@jorgebodega/typeorm-factory](https://www.npmjs.com/package/@jorgebodega/typeorm-factory)

***

### Factories :factory: 

> [src/seeding/factories]
I changed the Factory file in order to follow the new library syntax.

### Seeds :seedling: 

> [src/seeding/seeds]
You can still use seeding, but for now it's just hanging around doing NOTHING at all
 You can delete it if you wont implement seeding command

In case you will implement seeding, you will need to use another library which is this one:
[https://www.npmjs.com/package/@jorgebodega/typeorm-seeding](https://www.npmjs.com/package/@jorgebodega/typeorm-seeding)

### What is this PR code bringing

Basically, I implement the factory when the application starts, how?
Of course this is only testing, you don't want to keep creating users EVERY time your AppModule restarts

> [src/app.module.ts]

I will explicitly call the AppModule constructor, there I will be able to use the dataSource (Which you had set up before)

```
TypeOrmModule.forRoot({
		type: 'postgres',
		host: process.env.POSTGRES_CONTAINER,
		port: parseInt(process.env.POSTGRES_PORT_DOCKER),
		.... etc
```

Then I will call the factory and attempt to create some users from the factory, then I will save the users to the DB

# :magic_wand: 

***

### CODE REVIEW

**Comment by RodrigoEstrela** [comment]
Obrigado Andre!
